.github/workflows/publish.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     timeout-minutes: 15
     runs-on: ubuntu-latest
     container:
-      image: golang:bullseye
+      image: golang:1.19.0-bullseye
```

Comment:
how about we use `golang:1.19-bullseye`? That way we'll get the latest patch version on a build

Reply:
By using a specific Go version we know what exact version is used to build the binaries at any point in the Git/GHA history. For the most recent release (v0.4.15) although 1.18 was specified, per the CVE report we can see that version 1.18.4 was used. And, if version 1.19 wasn't available yet, we would had to upgrade to 1.18.5 to fix that CVE issue. We could go with 1.19, but I think using specific versions avoids surprises and makes things more explicit. Am I missing something?

Reply:
I suppose it would help to know the exact Go version used. My only concern with fixing the patch version is that there will be some effort required to make sure we're using the latest version with security and bug fixes. We would need a PR around once a month to keep that updated. Is that something we'll be fine with doing?

Reply:
I would agree with Rohit that it's better to let the patch version bump on its own and rely on tests to show breakage rather than manually bumping every patch version as CVEs come out.

Reply:
Ok, I'll remove the patch version. We can only add it when we need it for an upgrade. My guess is that we would still need to manually bump every patch version as CVEs come out because we would have very rare changes on this repo that would incidentally fix CVEs by building with the latest patch version.